src/commands/index.ts

```diff
@@ -1,21 +1,10 @@
 import * as vscode from 'vscode';
 import { getConfig } from '../utils/vscode-extensions';
 import { LanguageResourceManager } from '../i18n';
-import {
-  IconGenerator,
-  mergeConfig,
-  schema,
-  toggleAngularPreset,
-  toggleOfficialIconsPreset,
-  toggleHideFoldersPreset,
-} from '../icon-manifest';
+import * as iconManifest from '../icon-manifest';
 import { extensions as files } from '../icon-manifest/supportedExtensions';
 import { extensions as folders } from '../icon-manifest/supportedFolders';
-import {
-  IFileCollection,
-  IFolderCollection,
-  LangResourceKeys,
-} from '../models';
+import { IFileCollection, IFolderCollection, LangResourceKeys } from '../models';
```

Comment:
So I figured that if we reference up to three (3) imported members we can have it in one line. At least that's the guideline I'll follow.